css21/visudet/height-percentage-004.xht

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
+"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <title>CSS test: inherited percentage height</title>
+
+  <link rel="help" href="http://www.w3.org/TR/CSS22/visudet.html#the-height-property" title="10.5 Content height: the 'height' property" />
+  <meta name="assert" content="If the resulting height is smaller than 'min-height', the rules above are applied again, but this time using the value of 'min-height' as the computed value for 'height'. Note: These steps do not affect the real computed value of 'height'." />
+  <link rel="match" href="height-percentage-004-ref.xht" />
```

Comment:
This doesn't exist in this commit (though it's added later in `c90c4c42cc8fef3587897ba32eb90d35ef0d82fa`) if you care about having a clean history (it certainly makes reviewing easier!)